crp_APL/controllers/ctrl_vehicle_control_lat_pure_p/include/ctrl_vehicle_control_lat_pure_p/controller_inputs.hpp

```diff
@@ -0,0 +1,31 @@
+#ifndef CONTROLLER_INPUTS_HPP_
+#define CONTROLLER_INPUTS_HPP_
+
+
+#include <stdio.h>
+#include <string>
+#include <vector>
+
+
+namespace crp
+{
+    namespace apl
+    {
+        struct ControlInput{
+            std::vector<double> path_x;
```

Comment:
please use our way of noting m_ for member variables
```suggestion
            std::vector<double> m_path_x;
```